src/Analyzer.Core/Rules/BuiltInRules.json

```diff
@@ -394,7 +394,8 @@
           "path": "identity.type",
           "in": [
             "UserAssigned",
-            "SystemAssigned"
+            "SystemAssigned",
+            "SystemAssigned, UserAssigned"
```

Comment:
Reminder to consider the template might say "SystemAssigned,UserAssigned" (no space)

Reply:
Good catch, just added support and tests for this.

As a side-note: I did see that the ARM/bicep has a warning when the space is missing `Value must be one of the following values: "SystemAssigned", "UserAssigned", "SystemAssigned, UserAssigned", "None"`

Reply:
Based on the recent feedback, I will revert this change to only allow a single space.